src/dnx/dnx.cpp

```diff
@@ -243,6 +244,14 @@ int CallApplicationProcessMain(size_t argc, dnx::char_t* argv[], dnx::trace_writ
     // Set the DEFAULT_LIB environment variable to be the same directory as the exe
     SetEnvironmentVariable(_X("DNX_DEFAULT_LIB"), currentDirectory.c_str());
 
+	// Set the FRAMEWORK environment variable to the value provided on the command line
+	//  (it needs to be available BEFORE the application main is called)
+	auto frameworkName = GetOptionValue(argc, argv, _X("--framework"));
+	if (frameworkName)
+	{
+		SetEnvironmentVariable(_X("DNX_FRAMEWORK"), frameworkName);
```

Comment:
Ah, default C++ formatting in VS is tabs. That explains why it looked fine in VS :P

Fixing